engines/python/setup/djl_python/neuron_utils/model_loader.py

```diff
@@ -143,15 +144,26 @@ def set_neuronx_class(self):
             )
         return neuronx_class
 
+    def can_use_continuous_batching(self) -> bool:
+        """
+        Set configuration for continuous batching, currently all vllm implementations are continuous batching
+        and batch size greater than 1 for tnx and lmi-dist support rolling batch.
+
+        :return: bool indicating if continuous batching can be used
+        """
+        use_continuous_batching = (self.config.rolling_batch != "disable"
+                                   and self.config.rolling_batch_strategy
+                                   == TnXGenerationStrategy.continuous_batching
+                                   and self.config.max_rolling_batch_size
+                                   > 1) or self.config.rolling_batch == "vllm"
```

Comment:
does lmi-dist need to be handled here?

Reply:
Currently no, as of right now lmi-dist does not have this model compilation limitation, so this is base vLLM support only that requires continuous batching for batch size 1 (reason we don't use this is that the performance is worse compared to the alternative scheduler for batch size 1)